<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <node ns="pointcloud_repub" name="pointcloud_repub" pkg="pointcloud_republish" type="pointcloud_republish_node" output="screen">
      <param name="input_topic" value="/guidance/points2"/>
      <param name="output_topic" value="/guidance/points2_downsampled"/>
      <param name="leaf_size" value="0.05"/>
  </node>

</launch>
